ddl/ddl_worker.go

```diff
@@ -511,12 +511,26 @@ func (w *worker) handleDDLJobQueue(d *ddlCtx) error {
 	}
 }
 
+func skipWriteBinlog(job *model.Job) bool {
+	switch job.Type {
+	// ActionUpdateTiFlashReplicaStatus is a TiDB internal DDL,
+	// it's used to update table's TiFlash replica available status.
+	case model.ActionUpdateTiFlashReplicaStatus:
```

Comment:
also for ActionSetTiFlashReplica？

Reply:
`ActionSetTiFlashReplica` is not a TiDB internal DDL.